### ISSUE TITLE
Syntax error in inchlib_clust.py

### ISSUE THREAD

**Issue by anderev** [skutac/InCHlib.js#14]
Is it possible there are syntax errors in the inchlib_clust script? I'm getting the following error no matter how I change the command-line args:

```
python inchlib_clust.py heatmap_input.csv -dh -mh -cmh -d euclidean -l ward -a both -dd , -md
  File "inchlib_clust.py", line 401
    return {str(r[0]):r[1:] for r in alternative_data}
                              ^
SyntaxError: invalid syntax
```

**Comment by skutac** [comment]
Is it possible that you are using older python than 2.7? Dict comprehensions are available only in python 2.7 or higher.

**Comment by anderev** [comment]
Oh, right! Yes, it works in 2.7! Thank you!
